.github/workflows/conan.yml

```diff
@@ -40,6 +40,63 @@ jobs:
           github.event_name == 'schedule' &&
           steps.git_info.outputs.current_commit == steps.last_successful_commit.outputs.commit-hash
 
+  build_windows:
+    if: github.repository_owner == 'viamrobotics'
+    needs: [prepare]
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-windows
+            platform: windows_x86_64
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Install cmake
+        uses: ssrobins/install-cmake@v1
+        with:
+          version: 3.27.2
+
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install Scoop
+        uses: MinoruSekine/setup-scoop@v4.0.1
+
+      - name: install buf 'n stuff
+        run: scoop install buf ninja
+
+      - name: Install dependencies
+        uses: crazy-max/ghaction-chocolatey@v3
+        with:
+          args: install -y conan git 7zip jq
+
+      # Ensures that things installed with choco are visible to us
+      - name: refresh env
+        run: |
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
+
+      - name: Create package
+        shell: powershell
+        run: |
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
+          conan profile detect
+          conan create . `
+            --build=missing `
+            -o "&:shared=False" `
+            -s:h compiler.cppstd=17 `
+            -c:h tools.microsoft:winsdk_version=10.0.17763.0 `
+            -s:h compiler.runtime=static
```

Comment:
I wouldn't expect any of these four arguments to be needed for CI. For tflite_cpu they were required in order to get a static executable that would run on Windows 10 Build 1089 without requiring the VC redistributable, but those constraints are not in effect here.